index.js

```diff
@@ -50,7 +57,20 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies,rating) {
+  if(rating=== undefined){
+    rating = 'G'
+  }
+
+  if(movies.length === 0){
+    throw 'error'
+  }
+  
+  let rate = movies.some(
+    (movie)=> movie.rated === rating
+  );
+  return Boolean(rate)
```

Comment:
```suggestion
  return rate
```
rate is already a boolean, so you don't need to use Boolean here :)